go/cmd/berty/mini/view_group_outgoing.go

```diff
@@ -959,6 +959,11 @@ func newMessageCommand(ctx context.Context, v *groupView, cmd string) error {
 		return nil
 	}
 
+	// check for '/' messages
+	if strings.HasPrefix(cmd, "//") {
+		cmd = cmd[1:]
+	}
+
```

Comment:
It would be better if you fix the `newSlashMessageCommand` function in this file instead as it seems bugged.

I haven't tested it thoroughly, the following line as the function body should be enough.

```go
return newMessageCommand(ctx, v, fmt.Sprintf("/%s", cmd))
```

Reply:
I did and test, it seems to be working as expected :+1: